universal-application-tool-0.0.1/app/controllers/admin/AdminProgramBlocksController.java

```diff
@@ -46,6 +47,7 @@ public AdminProgramBlocksController(
     this.formFactory = checkNotNull(formFactory);
   }
 
+  /** Return a HTML page displaying all configurations of the program. */
```

Comment:
question: maybe add that this will show the block edit page for the last block in the program

Reply:
Done.